conf/app.routes

```diff
@@ -7,8 +7,11 @@ DELETE      /declarations/:id                                       uk.gov.hmrc.
 
 POST        /amendments                                             uk.gov.hmrc.exports.controllers.AmendmentController.create
 
-GET         /draft-declaration/:parentId                            uk.gov.hmrc.exports.controllers.DeclarationController.findOrCreateDraftFromParent(parentId: String)
-GET         /amend-declaration/:submissionId                        uk.gov.hmrc.exports.controllers.DeclarationController.findOrCreateDraftForAmend(submissionId: String)
+# Draft for both accepted and rejected amended declarations
+GET         /draft-for-amendment/:parentId/:enhancedStatus          uk.gov.hmrc.exports.controllers.DeclarationController.findOrCreateDraftFromParent(parentId: String, enhancedStatus: String, isAmendment: Boolean ?= true)
```

Comment:
U2Y: I'd prefer `amendment-draft` and `submission-draft` for the endpoint names, but not too bothered.

Reply:
I'm OK with the 1st one, but for the 2nd? Maybe `rejected-submission-draft`? Or something else that still suggests this is a draft for a **rejected** submission.